src/CacheServiceErrorMapper.ts

```diff
@@ -43,5 +43,7 @@ export function cacheServiceErrorMapper(
     case Status.ALREADY_EXISTS:
       return new AlreadyExistsError(err?.message);
   }
-  return new InternalServerError('unable to process request');
+  return new InternalServerError(
+    `unable to process request\n${err?.stack || ''}`
```

Comment:
why not make this same as line 27? any reason not to include message as well?

Reply:
Can we use a `cause` in our exceptions? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

Reply:
Hmm... maybe I'm not understanding this but it doesn't seem that `ErrorConstructor` does not have `options` or `cause` property?

<img width="894" alt="image" src="https://user-images.githubusercontent.com/22686260/159769838-ac90246d-fbe2-4638-b7aa-d46d14768fed.png">

Reply:
I implemented what Matt suggested below.

Reply:
just do matt's thing - I was only pushing back at the string formatting of a stack.

Reply:
👍